ludwig/models/gbm.py

```diff
@@ -0,0 +1,125 @@
+import copy
+from typing import Any, Dict, List, Tuple, Union
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torchmetrics
+
+from ludwig.constants import BINARY, LOGITS, MODEL_GBM, NAME, NUMBER
+from ludwig.features.base_feature import OutputFeature
+from ludwig.features.feature_utils import LudwigFeatureDict
+from ludwig.models.abstractmodel import AbstractModel
+from ludwig.utils import output_feature_utils
+
+
+class GBM(AbstractModel):
+    @staticmethod
+    def type():
```

Comment:
It looks like this could be an abstract static method in the AbstractModel base class.

```
@staticmethod
@abstractmethod
def type():
  raise NotImplementedError(...)
```

Reply:
7a69b9e